src/qml/qgismobileapp.qml

```diff
@@ -1251,7 +1297,8 @@ ApplicationWindow {
                    || dashBoard.activeLayer.geometryType() === Qgis.GeometryType.Point
         )
       state: qgisProject && qgisProject.topologicalEditing ? "On" : "Off"
-      iconSource: Theme.getThemeIcon( "ic_topology_white_24dp" )
+      iconSource: Theme.getThemeVectorIcon( "ic_topology_white_24dp" )
+      iconColor: "#ffffff"
```

Comment:
```suggestion
      iconColor: "white"
```

Reply:
@domi4484 , thanks for the review -- this issue will be fixed in the next PR :+1: